rpcs3/Emu/Cell/SPUASMJITRecompiler.cpp

```diff
@@ -1043,10 +1043,25 @@ void spu_recompiler::IRET(spu_opcode_t op)
 
 void spu_recompiler::BISLED(spu_opcode_t op)
 {
-	fmt::throw_exception("Unimplemented instruction" HERE);
+	extern asmjit::X86Mem t ;
```

Comment:
You can made this constant if you initialize it to value.

Reply:
is it possible inside mov instruction?

Reply:
@woj1993  check now

Reply:
I don't know.